apps/notepad/src/app/component/note-create/note-create.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component } from '@angular/core';
+import { Note } from '../notes-list/note.model';
+
+@Component({
+  selector: 'np-note-create',
+  templateUrl: './note-create.component.html',
+  styleUrls: ['./note-create.component.scss'],
+})
+export class NoteCreateComponent {
+  notesList: Note[] = [];
+
+  fetchStoredNotes() {
+    const initialList = localStorage.getItem('storelist');
```

Comment:
If notes are being fetched from localStorage at multiple places, then a common service file can be made for code reusability in multiple components.